unreleased.rst

```diff
@@ -18,3 +18,4 @@ HEAD — ongoing
 + [Ledger Api] *BREAKING CHANGE** In Protobuf ``Value`` message, rename ``decimal` field to ``numeric``.
 + [Sandbox] Fixed a bug that could lead to an inconsistent snapshot of active contracts being served
   by the ActiveContractsService under high load.
++ [DAML Standard Library] **BREAKING CHANGE** The ``(/)`` operator was moved out of the ``Fractional`` typeclass into a separate ``Divisible`` typeclass, which is now the parent class of ``Fractional``. The ``Int`` instance of ``Fractional`` is discontinued, but there is an ``Int`` instance of ``Divisible``. This change will break projects where a ``Fractional`` instance is defined. To fix it, add a ``Divisible`` instance and move the definition of ``(/)`` there.
```

Comment:
It will also break projects where a function was type-restricted to `Fractional` and used on `Int`. Otherwise, LGTM.

Reply:
Thanks! I added that in.